assets/questions/unknownAlphabet.js

```diff
@@ -0,0 +1,325 @@
+        "bea",
+        "bcc",
+        "bcd"
+      ],
+      allowedOutputs: ['ecdab']
+    }, {
+      input: [
+        "egdi",
+        "egdb",
+        "egej",
+        "egei",
+        "ebjj",
+        "ebjg",
+        "ebgh",
+        "ebgj",
+        "igac",
+        "igae",
+        "igia",
+        "igif",
+        "idcf",
+        "idcc",
+        "idh"
+      ],
+      allowedOutputs: ['afchjgdeib']
+    },
+    {
+      input: [
+        "curwfe",
+        "curwfe",
+        "curwst",
+        "curwsw",
+        "curuyc",
+        "curuyf",
+        "curuga",
+        "curugs",
+        "cuxdeh",
+        "cuxdeu",
+        "cuxdbv",
+        "cuxdbn",
+        "cuxolc",
+        "cuxolb",
+        "cuxojd",
+        "cuxojs",
+        "cpcnow",
+        "cpcnob",
+        "cpcnqu",
+        "cpcnqp",
+        "cpctkj",
+        "cpctkd",
+        "cpctfg",
+        "cpctfa",
+        "cpbjeq",
+        "cpbjec",
+        "cpbjz",
+        "cpbhx",
+        "cpbhc",
+        "khjzs",
+        "khjzu",
+        "khjis",
+        "khjiv",
+        "khfax",
+        "khfay",
+        "khftg",
+        "khftq",
+        "kmima",
+        "kmimd",
+        "kmiej",
+        "kmies",
+        "kmrbm",
+        "kmrbr",
+        "kmrus",
+        "kmrut"
+      ],
+      allowedOutputs: ['ljhmezirxygadoqckfsvntwbup']
+    }],
+    buggyOutputTests: [],
+    performanceTests: []
+  }, {
+    instructions: [
+      [
+        'Next, double-check your code to make sure it handles unexpected cases',
+        ', such as a test case with no solution or a test case with more than ',
+        'one correct answer.'
+      ].join(''),
+      [
+        'For the test case that has no solution, just return "". For the test ',
+        'case that has multiple answers, return the first answer in English ',
+        'lexicographic (alphabetical) order.'
+      ].join(''),
+      'Example dictionary:',
+      '[a, b, a]',
+      'Alphabet is: ""',
+      '[cba]',
+      'Alphabet is: "abc"'
+    ],
+    prerequisiteSkills: ['Topological Sorting', 'String Manipulation'],
+    acquiredSkills: ['String Manipulation'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'findAlphabet',
+    correctnessTests: [{
+      input: [
+        "a",
+        "b",
+        "a"
+      ],
+      allowedOutputs: ['']
+    }, {
+      input: [
+        "c",
+        "a",
+        "b",
+        "a"
+      ],
+      allowedOutputs: ['']
+    }, {
+      input: [
+        "cab",
+        "bc"
+      ],
+      allowedOutputs: ['acb']
+    }],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.failsOnNoSolution',
+      messages: [
+        [
+          'Try running your function on ["c", "a", "b", "a"] on paper. ',
+          'Is your result what you expect?'
+        ].join(''),
+        [
+          'It looks like your answer is not an empty string.',
+          'Can you examine whether your order is valid on the test case ',
```

Comment:
Rather than asking them the same test case, why add a third message explaining why their solution isn't working?